README.md

```diff
@@ -193,15 +193,15 @@ The `auth` option can be
 
    ```js
    const App = require('@octokit/app')
-   const Octokit = require('octokit')
+   const Octokit = require('@octokit/rest')
 
    const app = new App({ id: process.env.APP_ID, privateKey: process.env.PRIVATE_KEY })
    const octokit = new Octokit({
      async auth () {
        const installationAccessToken = await app.getInstallationAccessToken({ 
          installationId: process.env.INSTALLATION_ID 
        });
-       return `token ${installationAccessToken}`;
+       return `Bearer ${installationAccessToken}`;
```

Comment:
I’m pretty sure that here it should definitely be token

Reply:
You are rigth, but i guess it is also interchangable, I do have one app that uses `Bearer`. Anyway I updated to the correct form. Thanks

Reply:
`Bearer` is used for authentication as the GitHub app itself. `token` is used to authenticate as an app’s installation on a GitHub account or organization